docs/sources/features/datasources/influxdb.md

```diff
@@ -117,6 +121,61 @@ You can switch to raw query mode by clicking hamburger icon and then `Switch edi
 You can remove the group by time by clicking on the `time` part and then the `x` icon. You can
 change the option `Format As` to `Table` if you want to show raw data in the `Table` panel.
 
+## Flux support
+
+> Starting in v7.1 grafana will execute Flux queries in addition to InfluxQL queries
+
```

Comment:
Should we add something that is beta? @ryantxu I believe you said there is a beta flag for this somewhere now in Grafana.

Reply:
There is a very clear flag in the UI... not sure it is necessary in the docs.  I'm not sure what we do in general here, and think either wasy is fine:

![image](https://user-images.githubusercontent.com/705951/87328202-fd0af800-c4e9-11ea-8bb6-fba5effc408c.png)